spec/services/preserved_object_handler_confirm_version_spec.rb

```diff
@@ -124,6 +124,31 @@
         end
       end
 
```

Comment:
question:  for "confirm_version" do we now have the capacity to easily report when incoming version > catalog version + 1?    That is, if incoming is 4 and catalog is 2 ... does the code now support us detecting this case?  Asking because it pertains to ticket #451

Reply:
i don't think this work makes it easier (or harder).  we'd still have to differentiate the case where the version is greater, since we only look for whether the versions are equal or not in `confirm_version`.

and while i don't think this PR makes that differentiation easier in `check_existence` either, that differentiation is already easier in `check_existence` than in `confirm_version`, since `check_existence` already differentiates the `incomnig_version > pres_copy.version` case from other version mismatches (in the `incoming_version > pres_copy.version` condition in `check_existence`, we could just see whether the difference is greater than one, and return different result codes accordingly).